swagger_zipkin/zipkin_decorator.py

```diff
@@ -42,11 +44,15 @@ class ZipkinClientDecorator(object):
     :type client: :class:`swaggerpy.client.SwaggerClient` or :class:`bravado.client.SwaggerClient`.
     """
 
-    def __init__(self, client):
+    def __init__(self, client, context_stack=None):
```

Comment:
I think the default value needs to be ThreadLocalStack(), as otherwise we'd get errors in py_zipkin, right? If this is true then I wonder why this wasn't caught by any test.

Reply:
Seems like this is asking for an integration test :)